README.md

```diff
@@ -0,0 +1,74 @@
+# Cookiecutter PyPackage
+
+AICS Cookiecutter template for a Python package.
+
+## Features
+* Uses `tox` and `pytest` for local testing
```

Comment:
local testing is configured to use tox and pytest to trigger testing simply run tox from the command prompt in the project home folder.